src/report/controller/dtos.ts

```diff
@@ -0,0 +1,22 @@
+export class ReportDto {
```

Comment:
The Report is our Domain Object, representing the report from our internal business view. The ReportDto, is the representation we provide publicly over our api. This can be a identical, a subset of a Report or can even have some additional information, wich are use full for the api client.